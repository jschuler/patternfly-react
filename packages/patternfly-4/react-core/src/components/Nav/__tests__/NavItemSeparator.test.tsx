// Auto-generated do not edit

import * as React from 'react';
import { shallow } from 'enzyme';
import { NavItemSeparator } from '../NavItemSeparator';

describe('NavItemSeparator test', () => {
  it('NavItemSeparator should match snapshot', () => {
    const view = shallow(
      <NavItemSeparator
        className={"''"}
      />);
    expect(view).toMatchSnapshot();
  });
});
