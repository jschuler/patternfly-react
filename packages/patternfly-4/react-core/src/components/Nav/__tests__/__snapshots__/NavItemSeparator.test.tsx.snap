// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NavItemSeparator test NavItemSeparator should match snapshot 1`] = `
<li
  className="pf-c-nav__separator ''"
  role="separator"
/>
`;
